chat-plugins/helptickets.js

```diff
@@ -299,22 +330,26 @@ exports.pages = {
 					buf += `<p><Button>timerstalling</Button></p>`;
 					break;
 				case 'harassment':
+					if (!isLast) break;
 					buf += `<p>If someone is harassing you, click the appropriate button below and a global staff member will take a look. Consider using <code>/ignore [username]</code> if it's minor instead.</p>`;
 					buf += `<p>If you are reporting harassment in a battle, please save a replay of the battle.</p>`;
-					buf += `<p><button class="button" name="send" value="/helpticket submit PM Harassment">Report harassment in a private message (PM)</button> <button class="button" name="send" value="/helpticket submit Battle Harassment">Report harassment in a battle</button> <button class="button" name="send" value="/helpticket submit Chatroom Harassment">Report harassment in a chatroom</button></p>`;
+					buf += `<p><Button>confirmpmharrasment</Button> <Button>confirmbattleharrasment</Button> <Button>confirmchatharrasment</Button></p>`;
```

Comment:
``<button>``

Also did you mean to make these buttons not do anything?

Reply:
(sane thing later down, too)

Reply:
Theres a regexp at the bottom that transforms them, hence why its `<Button>`

Reply:
Yeah, maybe this should use a new different tag though

Reply:
It does, it uses a capital B. :|